packages/devextreme/js/viz/core/export.js

```diff
@@ -311,7 +342,28 @@ export let combineMarkups = function(widgets, options = { }) {
         return r;
     }, { items: [], rowOffsets: [], rowHeights: [], itemWidth: 0, totalHeight: 0, maxItemLen: 0, totalWidth: 0, backgroundColors: [] });
 
-    const backgroundColor = `data-backgroundcolor="${exportItems.backgroundColors.length === 1 ? exportItems.backgroundColors[0] : '' }" `;
+    const backgroundColor = `${exportItems.backgroundColors.length === 1 ? exportItems.backgroundColors[0] : '' }`;
+    const { totalWidth, totalHeight } = exportItems;
+
+    const rootElement = wrapItemsToElement(totalWidth,
```

Comment:
actually I expected moving of a `totalWidth` to the next line as well, but it's not so important, just for your information